terraform/deployments/github/repos.yml

```diff
@@ -110,6 +110,13 @@ repos:
       additional_contexts:
         - Test Ruby / Run RSpec
 
+  content-modelling-e2e:
+    required_status_checks:
+      standard_contexts: *standard_security_checks
+      additional_contexts:
+        - Lint / Eslint
+        - Prettier / Check
+
```

Comment:
Do you want to run the e2e tests as part of this repo?

Reply:
I did consider it, but I think we want to keep them only running on demand.

Reply:
Sure but how do you prevent someone merging in a bad test?

Reply:
Ah, that's fair. Have added 👍